core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/ccm/upgrade/CcmUpgradeEvent.java

```diff
@@ -0,0 +1,35 @@
+package com.sequenceiq.cloudbreak.core.flow2.cluster.ccm.upgrade;
+
+import com.sequenceiq.cloudbreak.reactor.api.event.cluster.upgrade.ccm.CcmRemoveAutoSshResult;
+import com.sequenceiq.cloudbreak.reactor.api.event.cluster.upgrade.ccm.CcmReregisterToClusterProxyResult;
+import com.sequenceiq.cloudbreak.reactor.api.event.cluster.upgrade.ccm.CcmUnregisterHostsResult;
+import com.sequenceiq.cloudbreak.reactor.api.event.cluster.upgrade.ccm.CcmUpgradeFailed;
+import com.sequenceiq.cloudbreak.reactor.api.event.cluster.upgrade.ccm.CcmUpgradePreparationFailed;
+import com.sequenceiq.cloudbreak.reactor.api.event.cluster.upgrade.ccm.CcmUpgradePreparationResult;
+import com.sequenceiq.flow.core.FlowEvent;
+import com.sequenceiq.flow.event.EventSelectorUtil;
+
+public enum CcmUpgradeEvent implements FlowEvent {
+    CCM_UPGRADE_EVENT("CCM_UPGRADE_TRIGGER_EVENT"),
```

Comment:
I do not see where this trigger event is being sent. :smile: It should have been probably coming from `StackCcmUpgradeService.upgradeCcm()`.

Reply:
It is used from the unit test. StackCcmUpgradeService is coming from an another commit, which wasn't in the master when I opened this pr

Reply:
no, it will be a different pr

Reply:
it will be a next pr